network-api/networkapi/wagtailpages/migrations/0022_update_wide_images.py

```diff
@@ -0,0 +1,72 @@
+# # Generated by Django 3.0.14 on 2021-08-10 18:19
```

Comment:
This file loops through pages in the mofo app, checks for any images that are set to "wide" and then updates the radio select field to be wide as well.